bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/handler/PulseaudioBridgeHandler.java

```diff
@@ -45,9 +47,10 @@
  * @author Tobias Bräutigam - Initial contribution
  *
  */
-public class PulseaudioBridgeHandler extends BaseBridgeHandler {
+public class PulseaudioBridgeHandler extends BaseBridgeHandler implements PulseAudioBindingConfigurationListener {
     private final Logger logger = LoggerFactory.getLogger(PulseaudioBridgeHandler.class);
 
+    @SuppressWarnings("null")
```

Comment:
I would not add it myself.

Reply:
OK. I have a tendency to tell my IDE to shut up about some silly warnings. I will have to restrain it ;)